client/src/actions/index.js

```diff
@@ -41,8 +44,18 @@ export const pageCounter = (counter, direction) => {
   }
 
   return { type: PAGE_COUNTER, payload: count };
-}
+};
 
 export const recordHistory = history => {
   return { type: CHECK_HISTORY, payload: history };
-}
+};
+
+export const emailSending = email => {
+  axios
+    .post('/api/send_email', email)
+    .then(res => console.log(res))
+    .catch(err => {
+      console.log('errorr while sending email', err);
```

Comment:
Again, instead of console logging there should be proper error management that returns a message to the user saying that something went wrong.

Reply:
Check hoslack branch "save reports"